src/CommunityCommons/javasource/communitycommons/Misc.java

```diff
@@ -670,4 +670,20 @@ public static boolean overlayPdf(IContext context, IMendixObject generatedDocume
         LOG.trace("Overlay PDF end");
         return true;
     }
+    
+    /**
+	 * Get the Cloud Foundry Instance Index (0 for leader and >0 for slave)
+	 * @return CF_INSTANCE_INDEX environment variable if available, otherwise -1
+	 */
+	public static long getCFInstanceIndex() {
+		long cfInstanceIndex = -1L;
+		
+		try {
+			cfInstanceIndex = Long.parseLong(System.getenv("CF_INSTANCE_INDEX"));
+		} catch(Exception e) {
+			return cfInstanceIndex;
```

Comment:
Please be specific in catching exceptions, so no generic catch-all Exception but for instance a multi-catch block. When something goes wrong here and -1 is returned, there is no way to know if that is caused by the `CF_INSTANCE_INDEX` environment variable not being present, or if it is present but not parseable as a Long. Also, the return statement in the catch-block is redundant because it's already at the end of the method. 

I suggest to log the exception to the CommunityCommons log node at INFO level, if you catch one.